tensorboard/webapp/plugins/npmi/views/violin_filters/violin_filter/violin_filter_component.scss

```diff
@@ -0,0 +1,40 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+@import 'tensorboard/webapp/theme/tb_theme';
+
+.chart-container {
+  background-color: #fff;
+  border-bottom: 1px solid mat-color($tb-foreground, border);
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+}
+
+.chart {
+  height: 300px;
+  width: calc(100%-20px);
```

Comment:
IIUC, `calc(100%-20px)` is invalid, and *must* be `calc(100% - 20px)`.
This is why Prettier doesn’t change it; they’re not equivalent.

Reply:
I’m marking this unresolved again because `calc(100%-20px)` is not what
you want. Quoth [MDN `calc()` § Notes][1]:

> The `+` and `-` operators **must be surrounded by whitespace**. For
> instance, `calc(50% -8px)` will be parsed as a percentage followed by
> a negative length — an invalid expression — while `calc(50% - 8px)` is
> a percentage followed by a subtraction operator and a length.
> Likewise, `calc(8px + -50%)` is treated as a length followed by an
> addition operator and a negative percentage.

[1]: https://developer.mozilla.org/en-US/docs/Web/CSS/calc#Notes

Reply:
I removed the call entirely as I don't need it. That's why I closed the comment.